README.md

```diff
@@ -3,7 +3,7 @@ Caffeine
 
 **CoArray Fortran Framework of Efficient Interfaces to Network Environments**
 
-Caffeine is a parallel runtime library that aims to support Fortran compilers with a programming-model-agnostic application binary interface (ABI) to various communication libraries.  Current work is on supporting the ABI with the [GASNet-EX] exascale-ready networking middleware.  Future plans include support for an alternative Message Passing Interface ([MPI]) back end.
+Caffeine is a parallel runtime library that aims to support Fortran compilers with a programming-model-agnostic application interface to various communication libraries.  Current work is on supporting the Parallel Runtime Interface for Fortran (PRIF) with the [GASNet-EX] exascale-ready networking middleware.  Future plans include support for an alternative Message Passing Interface ([MPI]) back end.
```

Comment:
Please make the PRIF text a hyperlink to the spec : https://doi.org/10.25344/S4DG6S

Please also add the PRIF spec at the bottom of the publication list, with complete citation:

Damian Rouson, Brad Richardson, Dan Bonachea, Katherine Rasmussen, "Parallel Runtime Interface for Fortran (PRIF) Design Document, Revision 0.2", Lawrence Berkeley National Laboratory Technical Report (LBNL-2001563), Dec 20 2023,
https://doi.org/10.25344/S4DG6S